views/includes/check-your-answers/beneficial-owner-individual.html

```diff
@@ -46,145 +46,173 @@
   {% endif %}
 {% endset %}
 
+{#  #}
```

Comment:
The {# #} are empty comments here, so i would add a comment just to highlight what is happening,
maybe 
`{# Build the rows array to be used by the govukSummaryList #}`

Reply:
Extended this comment to mention optional rows.